### PR TITLE
Fix and redesign sms characters count

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
We use additional fields bellow message textarea fields. Then characters count is little oversize and we made it little polish. 
Also fix one more issue

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create Sms message.
3. Save and close
3. Then try edit again
4. You should see 0 characters count
5. After this PR you should see characters count correctly
6. Also after this PR you should see characters count more decent and polish

image

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10838"></a>

### CODE REVIEW

File: app/bundles/SmsBundle/Views/Sms/form.html.php [hunk 1]
```diff
@@ -26,14 +26,10 @@
...
     <div class="col-md-12">
-        <?php echo $view['form']->row($form['message']); ?>
-        <div>
-            <?php echo $view['translator']->trans('mautic.sms.form.nbcharacter.counter'); ?>
-            <span id="sms_nb_char">0</span>
-        </div>
-        <div class="alert alert-info mt-10">
-            <?php echo $view['translator']->trans('mautic.sms.form.nbcharacter.infobox'); ?>
+        <div class="characters-count">
+            <label class="control-label" for="" data-toggle="tooltip" data-container="body" data-placement="top" title="" data-original-title="<?php echo $view['translator']->trans('mautic.sms.form.nbcharacter.infobox'); ?>">  <?php echo $view['translator']->trans('mautic.sms.form.nbcharacter.counter'); ?><span class="label label-default" id="sms_nb_char">0</span></label>
```
Comment (escopecz) [inline app/bundles/SmsBundle/Views/Sms/form.html.php]: Can this long line be re-formatted to something more readable?

**Review by escopecz — CHANGES_REQUESTED** [review]

File: app/bundles/SmsBundle/Views/Sms/form.html.php (hunk 1 above)
Comment (kuzmany) [reply app/bundles/SmsBundle/Views/Sms/form.html.php]: I reformatted it

**Review by escopecz — APPROVED** [review]
I see no issue in the code 👍

**Review by volha-pivavarchyk — APPROVED** [review]
Works well

**Pull request merged**